dynawo/sources/Common/DYNParameter.h

```diff
@@ -57,15 +57,18 @@ class ParameterCommon {
   ParameterCommon(const ParameterCommon& parameter);
 #endif
 
+#ifdef LANG_CXX11
   /**
-   * @brief Destructor
+   * @brief Default copy assigment operator
    */
-#ifdef LANG_CXX11
-  virtual ~ParameterCommon() = default;
-#else
-  virtual ~ParameterCommon() { }
+  ParameterCommon& operator=(const ParameterCommon&) = default;
```

Comment:
are we sure we need this copy assignement operator ? Usuually we havoid doing copies... better see if we can remove it alltogether

Reply:
We need an explicit definition because of this:

`error: definition of implicit copy assignment operator for 'ParameterCommon' is deprecated because it has a user-declared destructor [-Werror,-Wdeprecated-copy-dtor]`

We cannot have a deleted explicit definition because of this (in DYNDumpModel.cpp and DYNModelManager.cpp):

`error: object of type 'DYN::ParameterModeler' cannot be assigned because its copy assignment operator is implicitly deleted`
`note: copy assignment operator of 'ParameterModeler' is implicitly deleted because base class 'DYN::ParameterCommon' has a deleted copy assignment operator`